modules/node_modules/@ncigdc/components/CartDownloadButton.js

```diff
@@ -27,48 +37,118 @@ const styles = {
   },
 };
 
-const CartDownloadButton = ({
-  active,
-  setActive,
-  mouseDownHandler,
-  mouseUpHandler,
-  style,
-  theme,
-}) => (
-  <Button
-    style={style}
-    leftIcon={<DownloadIcon />}
-    rightIcon={<DownCaretIcon />}
-    onClick={() => setActive(true)}
-  >
-    Download
+const downloadCart = (files, dispatch) => {
+  const controlledFiles = files.filter((file) => file.access === 'controlled');
 
-    {active &&
+  if (controlledFiles.length) {
+    dispatch(setModal(
       <Column
-        style={{ ...dropdown, width: '22rem' }}
-        onMouseDown={mouseDownHandler}
-        onMouseUp={mouseUpHandler}
+        style={{ margin: '-15px' }}
       >
-        <Row style={styles.row(theme)}>
-          <DownloadIcon style={styles.iconSpacing} /> Manifest
-        </Row>
-        <Row style={styles.row(theme)}>
-          <DownloadIcon style={styles.iconSpacing} /> Cart
-        </Row>
+        <div className="modal-header">
+          <h4> Access Error </h4>
+        </div>
+        <div className="modal-body">
+          <p>You are attempting to download files that you are not authorized to access.</p>
+          <p>
+            <span className='label label-success'>
+              {files.length - controlledFiles.length}
+            </span> files that you are authorized to download.
+          </p>
+          <p>
+            <span className='label label-danger'>
+              {controlledFiles.length}
+            </span> files that you are not authorized to download.
+          </p>
+          <p>
+            Please <LoginButton />
+          </p>
+        </div>
+        <div
+          className="modal-footer"
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'flex-end',
+          }}
+        >
+          <Button
+            onClick={() => dispatch(setModal(null))}
+            style={{ margin: '0 10px' }}
+          >
+            <span>Cancel</span>
+          </Button>
+          <Button
+            onClick={() => downloadCart(files.filter((file) => file.access === 'open'), dispatch)}
+            style={{ margin: '0 10px' }}
+          >
+            <span>Download {files.length - controlledFiles.length} authorized files</span>
+          </Button>
+        </div>
       </Column>
+    ));
+  } else {
+    dispatch(setModal(null));
 
-    }
-  </Button>
-);
-
-CartDownloadButton.propTypes = {
-  active: PropTypes.bool,
-  setActive: PropTypes.func,
-  mouseDownHandler: PropTypes.func,
-  mouseUpHandler: PropTypes.func,
-  style: PropTypes.object,
+    download({
+      url: `${GDC_AUTH}api/data`,
+      params: {
+        ids: files.map((file) => file.file_id),
+      },
+      method: 'POST',
+      altMessage: true,
+    })(
+      () => {},
+      () => {}
```

Comment:
& match the download manifest button, also change the text to 'Downloading' when active